From a923a068ec8c44a88cfcac545d6d0cee074cbffa Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Fri, 20 May 2016 16:09:39 +0500
Subject: [PATCH] test/components: fix linking issue

The test binaries pick up incorrect rpaths due to the wrong
usage of linking of the libomxil-bellagio lib which
is produced by the same package.
We now use the libtool description file in the link
command to correctly identify and pick up the libs
without hardcoding.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
Upstream-status: Pending
---
 test/components/audio_effects/Makefile.am    | 4 ++--
 test/components/resource_manager/Makefile.am | 4 ++--

diff --git a/test/components/audio_effects/Makefile.am b/test/components/audio_effects/Makefile.am
index b28b992..ee9cffa 100644
--- a/test/components/audio_effects/Makefile.am
+++ b/test/components/audio_effects/Makefile.am
@@ -1,6 +1,6 @@
 check_PROGRAMS = omxvolcontroltest omxaudiomixertest
 
-bellagio_LDADD = -lomxil-bellagio
+bellagio_LDADD = $(top_builddir)/src/libomxil-bellagio.la
 common_CFLAGS = -I$(top_srcdir)/test/components/common -I$(includedir)
 
 omxvolcontroltest_SOURCES = omxvolcontroltest.c omxvolcontroltest.h
diff --git a/test/components/resource_manager/Makefile.am b/test/components/resource_manager/Makefile.am
index fcc1d99..c24d32c 100644
--- a/test/components/resource_manager/Makefile.am
+++ b/test/components/resource_manager/Makefile.am
@@ -1,6 +1,6 @@
 check_PROGRAMS = omxrmtest omxprioritytest
 
-bellagio_LDADD = -lomxil-bellagio
+bellagio_LDADD = $(top_builddir)/src/libomxil-bellagio.la
 common_CFLAGS = -I$(top_srcdir)/test/components/common -I$(includedir)
 
 omxrmtest_SOURCES = omxrmtest.c omxrmtest.h
-- 
1.9.1
